src/pages/verify/e-ids/norwegian-bankid.mdx

```diff
@@ -196,6 +196,7 @@ Biometrics is always included in the basis agreement.
 ### Existing customers
 
 Due to new terms and conditions in the agreement, you must send us an updated application if you wish to use BankID Biometrics.
+The application for Biometrics is identical as the one you will find below, but must also contain information about, whether your company is MVA obliged or not. 
```

Comment:
I markdown skal man have to linebreaks for at have et linjeskift i selve renderingen:

![image](https://user-images.githubusercontent.com/699049/188591576-d4d6795a-cdf9-4f51-9156-67fb3a9acb58.png)

Reply:
Så ved ikke om det er intentionen eller ej det skal være i samme paragraph @Winther9 ?

Reply:
Okay, æv! Troede lige jeg kunne uden hjælp eller fejl!

Reply:
Hvordan retter jeg det nu så? Skal du afslå og så går jeg ind igen, eller kan man gøre noget herfra?

Reply:
@Winther9

Tryk på "Files changed" i toppen og så her:

![image](https://user-images.githubusercontent.com/699049/188594408-8c253e9b-ebdc-457b-9498-da123b8b02a3.png)

Reply:
Tror den er der nu

Reply:
Yes :)